code/bit-array/src/test/java/ru/unn/agile/bitarray/test/BitArrayTest.java

```diff
@@ -0,0 +1,313 @@
+package ru.unn.agile.bitarray.test;
+
+import org.junit.Test;
+import ru.unn.agile.bitarray.model.BitArray;
+
+import static org.junit.Assert.*;
+
+public class BitArrayTest {
+
+    @Test
+    public void canInitBitArray() {
+        // arrange
+        BitArray btr = new BitArray(10);
+
+        // act
+        byte[] btrRawArray = btr.getRawArray();
+
+        // assert
+        assertEquals(btrRawArray.length, 2);
+    }
+
+    @Test(expected = NegativeArraySizeException.class)
+    public void canInitBitArrayWithNegative() {
+        new BitArray(-10);
+    }
+
+    @Test
+    public void isZeroAfterInitBitArray() {
+        // arrange
+        BitArray btr = new BitArray(101);
+
+        // act
+        byte[] btrRawArray = btr.getRawArray();
+
+        // assert
+        for (int elem : btrRawArray) {
+            assertEquals(elem, 0);
+        }
+    }
+
+    @Test
+    public void canUseGetCountBitFunc() {
+        // arrange
+        BitArray btr = new BitArray(279);
+
+        // act
+        int countBit = btr.getCountBit();
+
+        // assert
+        assertEquals(countBit, 279);
+    }
+
+    @Test
+    public void canSetBitInBitArray() {
```

Comment:
Can it be just simple test with one action?

Reply:
@Lenferd corrected